install-protobuf.sh

```diff
@@ -1,8 +1,10 @@
+#!/usr/bin/env sh
+
 set -ex
 
-basename=protobuf-2.6.0
-tarball=$basename.tar.bz2
+version=2.6.1
```

Comment:
we'd still have to update it to move the default along to keep up with latest. not sure i see the value